app/src/main/resources/explorer-backend-openapi.json

```diff
@@ -6689,17 +6689,19 @@
                 "schema": {
                   "type": "array",
                   "items": {
-                    "$ref": "#/components/schemas/TimedCount"
+                    "$ref": "#/components/schemas/TimedTxCount"
```

Comment:
`TimedCount` was originally meant to be a general case class that could be reused for any history data. But now that we introduce two count (total and non-coinbase), it's better to be explicit that it's for transactions count. It was only use there.